build_info/logcleaner.control

```diff
@@ -0,0 +1,10 @@
+Package: logcleaner
+Version: @DEB_LOGCLEANER_V@
+Architecture: @DEB_ARCH@
+Maintainer: @DEB_MAINTAINER@
+Depends: findutils, launchctl
```

Comment:
``launchctl`` as a dependency is unnecessary. You'll always have it on your system.

Reply:
i've seen people somehow not have launchctl

Reply:
It should be fine without it regardless. ``launchctl`` has been marked as essential, so there should be no issue.

Reply:
no reason not to have it.